### DISCUSSION TITLE
docs/nginx/

### DISCUSSION THREAD

**Discussion by giscus** [ricsanfre/pi-cluster#375] [Announcements]
# docs/nginx/

How to configure Nginx Ingress Controller in our Pi Kubernetes cluster.

https://picluster.ricsanfre.com/docs/nginx/

**Comment by i5Js** [comment]
There is a typo in the values file:

```
      - mountPath: /data
           name: data
```

Should be

```
      - mountPath: /data
        name: data
```

**Reply by ricsanfre** [reply]
Thanks @i5Js

Typo fixed

**Comment by ilbarone87** [comment]
Time for a migration here 😂
Likely would suggest to go for gateway-api, either cilium inbuilt gateway-api or k-gateway project.
Also traefik it's well maintained and supported by community.
